File: grokking-deep-learning/3_simple_neural_network.py
def neural_network(input_value: float, weight: float) -> float:
    return input_value * weight


def main():
    weight = 0.1
    number_of_toes = [8.5, 9.5, 10, 9]
    input_value = number_of_toes[0]
    prediction = neural_network(input_value, weight)
    print(prediction)


if __name__ == '__main__':
    main()

File: grokking-deep-learning/3_simple_neural_network.rs
fn main() {
    let weight = 0.1;
    let number_of_toes: [f64; 4] = [8.5, 9.5, 10., 9.];
    let input = number_of_toes[0];

    let prediction = neural_network(input, weight);
    println!("{}", prediction)
}

fn neural_network(input: f64, weight: f64) -> f64 {
    return input * weight
}